src/components/Annotation/index.js

```diff
@@ -4,6 +4,7 @@ import { annotationShape } from '../../utils/proptypes';
 const Annotation = ({ data, xScale, height, color, fillOpacity, id }) => (
   <rect
     key={id}
+    className={`id-${id}`}
```

Comment:
`id-foo` seems like not a very descriptive (or unique) class name. What do you think about something like:
```
className={`griff-annotation griff-annotation-${id}`}
```